packages/nextjs/pages/index.tsx

```diff
@@ -209,37 +217,125 @@ const Home: NextPage<{ stats: Stats }> = ({ stats }) => {
+                className="btn btn-primary btn-md px-8 mt-8"
               >
-                Connect with us
+                Co-fund with us
               </TrackedLink>
-              <p className="lg:mb-3 mt-12 lg:mt-8 text-sm">SUPPORTED BY</p>
-              <div className="flex flex-col gap-6 items-center lg:items-start">
-                <Image src="/assets/ef-logo.png" alt="EF logo" width={200} height={200} />
-                <div className="flex flex-col md:flex-row gap-7 items-center">
-                  <Image src="/assets/op-logo.svg" alt="Optimism logo" width={30} height={30} />
-                  <Address
-                    address="0x34aA3F359A9D614239015126635CE7732c18fDF3"
-                    size="sm"
-                    className="bg-gray-200 p-1 px-2 rounded-lg font-medium"
-                  />
+            </div>
+            {/* Cohorts Table */}
+            <div className="hidden xs:block mt-0 lg:mt-8">
+              <table className="min-w-full">
+                <thead>
+                  <tr className="text-base">
+                    <th className="bg-base-100 text-left py-3 px-4 xl:px-8">Name</th>
+                    <th className="bg-base-100 text-left py-3 px-4 xl:px-8">Hackers</th>
+                    <th className="bg-base-100 text-left py-3 px-4 xl:px-8">Balance</th>
+                  </tr>
+                </thead>
+                <tbody className="shadow-custom rounded-3xl text-sm">
+                  {cohortsData?.map(cohort => (
+                    <tr
+                      className="bg-base-400 hover:bg-base-100 border-b border-base-100 cursor-pointer"
+                      key={cohort.id}
+                      onClick={() => window.open(cohort.url, "_blank")}
+                    >
+                      <td className="py-3 px-4 xl:px-8">{cohort.name}</td>
+                      <td className="py-3 px-4 xl:px-8">{Object.keys(cohort.builders).length}</td>
+                      <td className="py-3 px-4 xl:px-8">
+                        {parseFloat(cohort.balance).toFixed(2)}
+                        <span className="text-xs ml-1">ETH</span>
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              </table>
+              {/* Badges with Cohorts table data aggregation*/}
+              <div className="mt-4 flex gap-4">
+                <div className="badge badge-primary font-normal border-opacity-20 bg-opacity-20 py-3 px-4">
+                  Cohorts <span className="ml-2 font-bold">{cohortsData?.length}</span>
+                </div>
+                <div className="badge badge-primary font-normal border-opacity-20 bg-opacity-20 py-3 px-4">
+                  Hackers{" "}
+                  <span className="ml-2 font-bold">
+                    {
+                      // Calculation for Total Number of Hackers
+                      cohortsData?.reduce((acc, cohort) => acc + Object.keys(cohort.builders).length, 0)
```

Comment:
Maybe we can do this calculation in `getStaticProps` and pass in `totalHackers` : 

```
// getStaticProps
  return {
    props: {
      stats,
      cohortsData,
      totalHackers,
    },
    // 6 hours refresh.
    revalidate: 21600,
  };


```

Reply:
Nice suggestion! I've just updated all calculations to do them in `getStaticProps` 🙌